<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1131</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QLabel" name="Sig1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>31</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Sig2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>31</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Sig3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>530</y>
      <width>31</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="rev">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>530</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="send">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>600</y>
      <width>501</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="gb">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>491</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Selective Robot</string>
    </property>
    <widget class="QComboBox" name="cbRobot">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>211</width>
       <height>32</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="goStop">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Go</string>
     </property>
    </widget>
    <widget class="QLabel" name="stream">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>471</width>
       <height>401</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>461</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>In manual mode, drive this robot by a PS4 controller</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="allCars">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>530</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>All Cars</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="manual">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="automatic">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Automactic</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goAll">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>100</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Go all</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopAll">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>100</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop all</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
